README.md

```diff
@@ -420,6 +443,22 @@ firewall:
     state: disabled
 ```
 
+Allow interface named '8086:15d7' in dmz zone
```

Comment:
Not sure I understand this example -- are there circumstances where the interface name would be a PCI device ID?

Reply:
Firewalld supports adding interfaces by any name, if it doesn't match with any logical interface name, it still gets added as a firewall rule without going through NetworkManager, but it likely will not match with any traffic. Since firewalld supports this, I felt it would be better practice to support it as well, even if it is just for removing interface names that do not match up with any network traffic.

Reply:
Thanks for the context.  I would suggest that we mention this info you provided under the "interface" section of the README and remove this particular example from the README as it sounds like this example is not something that would generally be used or recommended, and the example might cause some confusion (it did for me at least).  What are your thoughts on this?

Reply:
I agree, if it mislead you, I should change how the information is presented. I will move this information over to the interface section as a warning, since it is something the general user would need to know about when adding/moving interfaces.

Reply:
Updated README, should the new warning under `interface` be visible enough to users?

Reply:
This looks good to me and is very easy to understand.  Thanks!